github-pullrequest-plugin/src/test/java/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPRLabelExistsEventTest.java

```diff
@@ -0,0 +1,228 @@
+package org.jenkinsci.plugins.github.pullrequest.events.impl;
+
+import hudson.model.TaskListener;
+import org.jenkinsci.plugins.github.pullrequest.GitHubPRCause;
+import org.jenkinsci.plugins.github.pullrequest.GitHubPRLabel;
+import org.jenkinsci.plugins.github.pullrequest.GitHubPRPullRequest;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kohsuke.github.*;
```

Comment:
this is just copied code from other event, I have no idea what this does or it should do